Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -44,7 +44,8 @@ class LoginsHelper: BrowserHelper {
     }
 
     func userContentController(userContentController: WKUserContentController, didReceiveScriptMessage message: WKScriptMessage) {
-        var res = message.body as! [String: String]
+        // Normally [String: String] would work but Swift 2.0 seems to have issues converting NSString->String implicitly from the WKScriptMessage
+        guard var res = message.body as? [NSString: NSString] else { return }
```

Comment:
Wonder if we should throw an `assertionFailure` in here so we can break and catch this on debug builds? That's assuming this should never happen, which it doesn't seem like it should.